web-client/example/module.js

```diff
@@ -44,6 +44,11 @@ init().then(async () => {
         document.querySelector('#peers').textContent = numPeers;
     });
 
+    document.querySelector('#address-book').addEventListener("click", async () => {
+        let contacts = await client.getAddressBook();
+        console.log(contacts);
```

Comment:
```suggestion
        console.table(contacts);
```

Utilising `console.table` here makes sense as it's a list.